src/main/java/de/bonndan/nivio/output/dld4e/Dld4eRenderer.java

```diff
@@ -14,6 +15,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+@Deprecated //this is both layouter and renderer
```

Comment:
This was an early approach to use dld4e (see https://github.com/bonndan/dld4e-headless) to draw landscapes, but it lacked automatic layouting. It was kept in case it could become a plugin (#14) or somebody might be interested to use it with the automatic layouting. For now I'd say there is no replacement.

Reply:
So, it is deprecated but we have no replacement? 

I still don't understand what to do about the Dld4eRenderer :) If it is deprecated it means that we don't want it to be used anymore. We should add a comment on what to use instead.

Otherwise, if we want to keep it, it should not be deprecated.

Reply:
Actually I am too lazy to reimplement dld4e rendering with with new layout, although it might not take too long. I know it's in the repo, but I'm still afraid it will be forgotten if we'd delete it. If we're strict then it should be deleted.

Reply:
Mhm, well you decide. My opinion is: get rid of it. If it is important, we have the VCS to get the code back ;)

Reply:
Otherwise, it wasn't important 😸

Reply:
Removed it. :crying_cat_face: